superset/__init__.py

```diff
@@ -230,7 +230,11 @@ def is_feature_enabled(feature):
 if conf.get('ENABLE_FLASK_COMPRESS'):
     Compress(app)
 
-Talisman(app, content_security_policy=None)
+if app.config['TALISMAN_ENABLED']:
+    talisman_config = app.config.get('TALISMAN_CONFIG') or {
+        'content_security_policy': None,
```

Comment:
@craig-rueda any reason to have `{'content_security_policy': None}` as a fallback? Wouldn't one simply specify the options they require in their `superset_config.py` file?

Reply:
That was the setting that was there before - assumed it had a purpose

Reply:
This is to match the current behavior by default (`Talisman(app, content_security_policy=None)`)

Reply:
@mistercrunch wouldn't there be merit in having the logic in `superset/__init__.py` be, 
```
TALISMAN_CONFIG = {
    'content_security_policy': None,
}
```
to match the default behavior? In this example it seems like @craig-rueda want's to have `force_https_permanent` to be `True` which should then be overridden in the `superset_config.py` file.